istio/istio-install/base/istio-noauth.yaml

```diff
@@ -14415,6 +14422,13 @@ metadata:
     app: istio-egressgateway
     istio: egressgateway
 spec:
+  selector:
+    matchLabels:
+      chart: gateways
+      heritage: Tiller
```

Comment:
nit: can we get rid of the heritage label in the selector?

Reply:
@jlewi I did not want to break anything. The labels, above an below the selectors have to match. Those currently contain Tiller everywhere. You will have to find somebody to run a script and do "grep -v Tiller" on the entire directory structure. We should do that in another PR.

Reply:
@jlewi Removal of Tiller has been done incrementally in this PR #378